workbench-core/environments/src/postDeployment/cognitoSetup.ts

```diff
@@ -108,31 +111,19 @@ export default class CognitoSetup {
    * @returns user pool id
    */
   public async getUserPoolId(): Promise<string | undefined> {
```

Comment:
might be out of scope for this PR. do we have a unit test for this method?

Reply:
it's within the scope of this PR. Good catch